docs/en/latest/router-radixtree.md

```diff
@@ -289,6 +291,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
 }'
 ```
 
+We can verify GraphQL matches in the following two ways:
+
+1. GraphQL query strings
+
+    ```shell
+    $ curl -H 'content-type: application/graphql' -X POST http://127.0.0.1:9080/graphql -d '
+    query getRepo {
+        owner {
+            name
+        }
+        repo {
+            created
+        }
+    }'
+    ```
```

Comment:
Remove indentation, formatting needs to be consistent across the document.

Reply:
OK